core/src/main/java/io/grpc/ClientInterceptors.java

```diff
@@ -141,7 +141,9 @@ public void cancel() { }
     public void halfClose() { }
 
     @Override
-    public void sendPayload(Object payload) { }
+    public boolean sendPayload(Object payload) {
+      return true;
```

Comment:
Oh ok ... the detail you were leaving out (and the source of my confusion) was that this is only used when an exception occurs in the `start` method ... meaning the call is closed.  So in this case it makes sense to always return `false`.